coalib/bearlib/abstractions/Linter.py

```diff
@@ -207,6 +207,9 @@ def check_prerequisites(cls):
             :return:
                 True if operational, otherwise a string containing more info.
             """
+            if not super(LinterBase, cls).check_prerequisites():
+                return False
+
```

Comment:
@arush0311 It's fine like it is... almost ;)

@Siddhartha1234 You can reduce `super(LinterBase, cls)` to just `super()`